packages/instant/src/style/z_index.ts

```diff
@@ -2,5 +2,6 @@ export const zIndex = {
     errorPopBehind: 1,
     mainContainer: 2,
     panel: 3,
-    errorPopUp: 4,
+    containerOverlay: 4,
```

Comment:
This will have conflicts with https://github.com/0xProject/0x-monorepo/pull/1230/